src/Sentry.Hangfire/Sentry.Hangfire.csproj

```
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Official Hangfire integration for Sentry - Open-source error tracking that helps developers monitor and fix crashes in real time.</Description>
+    <PackageTags>$(PackageTags);OpenTelemetry</PackageTags>
+    <TargetFrameworks>net8.0;net6.0;netstandard2.1;netstandard2.0;net462</TargetFrameworks>
```

We can always add things later, how about we go with only this to start:
```suggestion
    <TargetFrameworks>net8.0;net6.0;net462</TargetFrameworks>
```

---

This was resolved but we ended up with both netstandard TFMs, is that by design?